up-linux-streamer/src/main.rs

```diff
@@ -0,0 +1,66 @@
+use log::trace;
+use std::fs::canonicalize;
+use std::path::PathBuf;
+use std::sync::Arc;
+use std::thread;
+use up_rust::UStatus;
+use up_rust::UTransport;
+use up_streamer::{Endpoint, UStreamer};
+use up_transport_vsomeip::UPTransportVsomeip;
+use up_transport_zenoh::UPClientZenoh;
+use zenoh::config::Config;
+
+#[tokio::main]
+async fn main() -> Result<(), UStatus> {
+    env_logger::init();
+
+    let mut streamer = UStreamer::new("up-linux-streamer", 10000);
+
+    let crate_dir = env!("CARGO_MANIFEST_DIR");
+    // TODO: Make configurable to pass the path to the vsomeip config as a command line argument
+    let vsomeip_config = PathBuf::from(crate_dir).join("vsomeip-configs/point_to_point.json");
+    let vsomeip_config = canonicalize(vsomeip_config).ok();
+    trace!("vsomeip_config: {vsomeip_config:?}");
+
+    // There will be a single vsomeip_transport, as there is a connection into device and a streamer
+    // TODO: Add error handling if we fail to create a UPTransportVsomeip
+    let vsomeip_transport: Arc<dyn UTransport> = Arc::new(
+        UPTransportVsomeip::new_with_config(&"linux".to_string(), 10, &vsomeip_config.unwrap())
+            .unwrap(),
+    );
+
+    // TODO: Probably make somewhat configurable?
+    let zenoh_config = Config::default();
+    // TODO: Add error handling if we fail to create a UPClientZenoh
+    let zenoh_transport: Arc<dyn UTransport> = Arc::new(
+        UPClientZenoh::new(zenoh_config, "linux".to_string())
+            .await
+            .unwrap(),
+    );
+    // TODO: Make configurable to pass the name of the mE authority as a  command line argument
+    let vsomeip_endpoint = Endpoint::new(
+        "vsomeip_endpoint",
+        "me_authority",
```

Comment:
my understanding is that this is the logical name of one of the MCUs that we want to interact with and that if another uE on the zenoh transport wants to invoke a method on that MCU it would use `me_authority` in the sink UUri of the RPC Request message?

Reply:
Currently the vsomeip transport supports only a single mDevice, but given the nature of how I must use vsomeip with one vsomeip application per uE on the host, I'm loathe to have to spin up multiple UPTransportVsomeip.

Given that, I've started to poke around with @stevenhartley on [this comment](https://github.com/eclipse-uprotocol/up-transport-vsomeip-rust/pull/4#discussion_r1679327915) to see if we can't make good use of the instance id to have a single logical name for the entire SOME/IP network.

Reply:
Marking as resolved for now, but should be followed up on in:
https://github.com/eclipse-uprotocol/up-transport-vsomeip-rust/issues/9